data/settings/1.15.x/ntp.toml

```
@@ -0,0 +1,7 @@
+[[docs.ref.time-servers]]
+description = """
+A list of NTP servers to set and verify the system time.
+"""
+accepted_values = [ "hostnames", "IP addresses" ]
+[[docs.ref.time-servers.example]]
+value = "[\"169.254.169.123\", \"2.amazon.pool.ntp.org\"]"
```

We're likely to change this in the future, so consider suggesting it now:

```suggestion
[[docs.ref.time-servers.example]]
value = "[\"169.254.169.123\", \"time.aws.com\"]"
```

(Or `time.google.com`, it is less about the vendor and more about the consistency of time sources with respect to leap seconds.)